### PR TITLE
#30541 adding the resetable to the workflow action response

### LINKED ISSUE #30541: ContentEditor: Add resetable property to the workflow action
### Parent Issue

_No response_

### User Story

As an user I want to be able to reset the workflow, in order to do it we need to provide a way to the UI to know if the action has or not a reset actionlet in any of the actions
So we will return a boolean property called isResetable that will tells if the actionlet can or not be used for reset

### Acceptance Criteria

Actionlet with a reset sub actionlet assign will set the isResetable in true

### Proposed Objective

Core Features

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by jdotcms**
If the action has a reset actionlet, the resetable property will be true

This PR fixes: #30541

### CODE REVIEW

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 1]
```diff
@@ -17775,6 +17775,199 @@
...
+					"response": []
+				},
+				{
+					"name": "CheckResetable",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"exec": [
+									"// Test para verificar que la respuesta tiene el estado 200 OK",
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:17860]: Translate to English

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 2]
```diff
@@ -17775,6 +17775,199 @@
...
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"// Test para verificar que el JSON de respuesta tiene las propiedades principales",
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:17865]: Translate to English

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 3]
```diff
@@ -17775,6 +17775,199 @@
...
+									"    const jsonData = pm.response.json();",
+									"    pm.expect(jsonData).to.have.property(\"entity\");",
+									"    pm.expect(jsonData).to.have.property(\"errors\");",
+									"    pm.expect(jsonData).to.have.property(\"i18nMessagesMap\");",
+									"    pm.expect(jsonData).to.have.property(\"messages\");",
+									"    pm.expect(jsonData).to.have.property(\"pagination\");",
+									"    pm.expect(jsonData).to.have.property(\"permissions\");",
+									"});",
+									"",
+									"// Test para verificar que \"entity\" es un array y tiene al menos un elemento",
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:17876]: English

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 4]
```diff
@@ -17775,6 +17775,199 @@
...
+									"pm.test(\"Entity is an array and contains items\", function () {",
+									"    const entity = pm.response.json().entity;",
+									"    pm.expect(entity).to.be.an(\"array\").that.is.not.empty;",
+									"});",
+									"",
+									"// Test para verificar las propiedades de los objetos dentro de \"entity\"",
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:17882]: English

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 5]
```diff
@@ -17775,6 +17775,199 @@
...
+									"        pm.expect(item).to.have.property(\"nextStep\").that.is.a(\"string\");",
+									"        pm.expect(item).to.have.property(\"nextStepCurrentStep\").that.is.a(\"boolean\");",
+									"        pm.expect(item).to.have.property(\"order\").that.is.a(\"number\");",
+									"        pm.expect(item).to.have.property(\"owner\").that.is.null;",
+									"        pm.expect(item).to.have.property(\"hasResetActionlet\").that.is.a(\"boolean\");",
+									"        pm.expect(item).to.have.property(\"roleHierarchyForAssign\").that.is.a(\"boolean\");",
+									"        pm.expect(item).to.have.property(\"schemeId\").that.is.a(\"string\");",
+									"        pm.expect(item).to.have.property(\"showOn\").that.is.an(\"array\");",
+									"        ",
+									"        // Verificación adicional: que \"showOn\" contiene valores específicos esperados",
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:17914]: English

File: dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json [hunk 6]
```diff
@@ -17775,6 +17775,199 @@
...
+									"        const allowedShowOnValues = [\"NEW\", \"LISTING\", \"PUBLISHED\", \"UNLOCKED\", \"ARCHIVED\", \"UNPUBLISHED\", \"EDITING\", \"LOCKED\"];",
+									"        item.showOn.forEach(status => {",
+									"            pm.expect(allowedShowOnValues).to.include(status);",
+									"        });",
+									"    });",
+									"});",
+									"",
+									"// Test para verificar que \"errors\" es un array vacío",
```
Comment (dsilvam) [inline dotcms-postman/src/main/resources/postman/Workflow_Resource_Tests.json:17922]: English

**Review by jcastro-dotcms — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Pull request merged**